ios-showcase-template.xcodeproj/project.pbxproj

```diff
@@ -918,15 +922,17 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = "ios-showcase-template/ios-showcase-template.entitlements";
+				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = GHPBX39444;
 				INFOPLIST_FILE = "ios-showcase-template/Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = "org.aerogear.ios-showcase-template";
+				PRODUCT_BUNDLE_IDENTIFIER = org.aerogear.ShowcaseTemplate;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE = "c708fd4a-a5c7-4353-b315-5d70039e67ce";
-				PROVISIONING_PROFILE_SPECIFIER = RHTINHOUSE;
+				PROVISIONING_PROFILE = "";
```

Comment:
Another important change is to not lock on provisioning profile. Xcode will pick that up automatically depending if dev has created his own developer account or has access to internal one etc.